res/res_config_pgsql.c

```diff
@@ -1459,61 +1459,66 @@ static int parse_config(int is_reload)
 		pgsqlConn = NULL;
 	}
 
-	if (!(s = ast_variable_retrieve(config, "general", "dbuser"))) {
-		ast_log(LOG_WARNING,
-				"PostgreSQL RealTime: No database user found, using 'asterisk' as default.\n");
-		strcpy(dbuser, "asterisk");
-	} else {
-		ast_copy_string(dbuser, s, sizeof(dbuser));
-	}
-
-	if (!(s = ast_variable_retrieve(config, "general", "dbpass"))) {
-		ast_log(LOG_WARNING,
-				"PostgreSQL RealTime: No database password found, using 'asterisk' as default.\n");
-		strcpy(dbpass, "asterisk");
-	} else {
-		ast_copy_string(dbpass, s, sizeof(dbpass));
-	}
-
-	if (!(s = ast_variable_retrieve(config, "general", "dbhost"))) {
-		ast_log(LOG_WARNING,
-				"PostgreSQL RealTime: No database host found, using localhost via socket.\n");
-		dbhost[0] = '\0';
-	} else {
-		ast_copy_string(dbhost, s, sizeof(dbhost));
-	}
-
-	if (!(s = ast_variable_retrieve(config, "general", "dbname"))) {
-		ast_log(LOG_WARNING,
-				"PostgreSQL RealTime: No database name found, using 'asterisk' as default.\n");
-		strcpy(dbname, "asterisk");
-	} else {
-		ast_copy_string(dbname, s, sizeof(dbname));
-	}
-
-	if (!(s = ast_variable_retrieve(config, "general", "dbport"))) {
-		ast_log(LOG_WARNING,
-				"PostgreSQL RealTime: No database port found, using 5432 as default.\n");
-		dbport = 5432;
-	} else {
-		dbport = atoi(s);
-	}
-
-	if (!(s = ast_variable_retrieve(config, "general", "dbappname"))) {
-		dbappname[0] = '\0';
-	} else {
-		ast_copy_string(dbappname, s, sizeof(dbappname));
-	}
-
-	if (!ast_strlen_zero(dbhost)) {
-		/* No socket needed */
-	} else if (!(s = ast_variable_retrieve(config, "general", "dbsock"))) {
-		ast_log(LOG_WARNING,
-				"PostgreSQL RealTime: No database socket found, using '/tmp/.s.PGSQL.%d' as default.\n", dbport);
-		strcpy(dbsock, "/tmp");
-	} else {
-		ast_copy_string(dbsock, s, sizeof(dbsock));
-	}
+    if ((s = ast_variable_retrieve(config, "general", "dbuser"))) {
+        ast_log(AST_LOG_WARNING, "Option 'dbuser' has been deprecated in favor of 'user'. Please make that change in your res_pgsql.conf.\n");
```

Comment:
I don't see any benefit to deprecating the old configuration option names. Supporting both is fine for consistency, but deprecating the old ones for the sake of deprecating them in the name of consistency doesn't sit well with me.

Reply:
Should we support both in the other modules ?

Reply:
I don't think so - one should just be picked and stuck with.

Reply:
got it.